Update TagCellLayout.swift

spacecaps-cao opened the pull request:
Add a variable called "lineNumber" to record how many lines of tags view. So that users can calculate the contentSize of UICollectionView dynamically and change the constraints of UICollectionView. (In the case of this UICollectionView is embeded in other views or animationing)

Commit:
Update TagCellLayout.swift

Add a variable called "lineNumber" to record how many lines of tags view. So that users can calculate the contentSize of UICollectionView dynamically and change the constraints of UICollectionView. (In the case of this UICollectionView is embeded in other views or animationing)

riteshhgupta commented:
@Cao-Yu thanks for the PR & it looks good but just one thing, you forgot to reset the `lineNumber` in `resetLayoutState()` method. Can you please do that as well ?

Commit:
Update TagCellLayout.swift

Reset the lineNumber in resetLayoutState() method. Sorry for my carelessness. This is my first PR, Thank you! XD
